test/test_readers.py

```diff
@@ -74,3 +74,24 @@ def test_local_recursive_nested(tmpdir):
     make(tmpdir, filenames)
     actual = LocalFileReader().list(str(tmpdir), recursive=True)
     assert parse(actual) == expected
+
```

Comment:
I tried moving them but I would need to either import the `make` and `parse` functions or copy paste them into `test_images_io.py`. I think they belong in the `test_readers.py` that is the main function that I changed.